### DISCUSSION TITLE
调解亮度失效

### DISCUSSION THREAD

**Discussion by smarttigerboy** [shuax/MouseInc.Settings#439] [General]
2.10.21版本可正常使用，升级2.13.4后失效。不知什么原因。

鼠标滚轮在屏幕四个边滚动、按下时触发的功能。
 位置 |      名称      |             向上动作                        |                 向下动作                    |            按下动作
  左    | 亮度调节   |       [ "SetBrightness", "Up" ]       |     [ "SetBrightness", "down" ]    |       [ "SetBrightness", "DEFAULT" ]

**Comment by smarttigerboy** [comment]
2.10.21版本 和2.13.4版本 调节亮度底层执行的方式不同吗

**Reply by Ctrl-Ling** [reply]
是的 执行方式修改了，但主要还是新的调节方式不支持扩展屏幕导致，可以参考这个讨论https://github.com/shuax/MouseInc.Settings/discussions/300
